frontend/src/components/NavBarLink.js

```diff
@@ -48,15 +48,15 @@ class NavBarLink extends Component {
     if (hardReload) {
       return (
```

Comment:
as codeclimate pointed out (and I was thinking as I coded it) I guess the code duplication in this file kinda sucks. If you see an obvious way to refactor @devvmh would welcome it